### PR TITLE
gobject-introspection 1.66.1

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **gobject-introspection/1.66.1**

try FreeBSD

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/gobject-introspection/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
```
Comment (uilianries) [inline recipes/gobject-introspection/all/conanfile.py:34]:
```suggestion
            del self.options.fPIC

    def validate(self):
```

Comment (ericLemanissier) [reply recipes/gobject-introspection/all/conanfile.py:34]: I had very strange bugs on CCI when using validate(self) the last days:
eg https://github.com/conan-io/conan-center-index/pull/4181#issuecomment-756993143 or https://github.com/conan-io/conan-center-index/pull/4182#issuecomment-757434081
so I'm not very inclined to use it until we understand what happened.  Can we wait to introduce it ?

Comment (prince-chrismc) [reply recipes/gobject-introspection/all/conanfile.py:34]: #4193 too

**Review by prince-chrismc — APPROVED** [review]

File: recipes/gobject-introspection/all/conanfile.py (hunk 1 above)
Comment (theirix) [reply recipes/gobject-introspection/all/conanfile.py:34]: Fair enough, `validate` is experimental and even missing from the Conan changelog :) Hope we would refactor recipes to use `validate` later.

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**